.eslintrc.json

```diff
@@ -30,7 +30,8 @@
     ".idea/",
     ".vscode/",
     "package-lock.json",
-    "yarn.lock"
+    "yarn.lock",
+    "src/*"
```

Comment:
shoudn't be ignoring `src/*`

Reply:
I ignore while developing.  These rules should be applied precomit to improve the development experience